src/components/Form/FormFields/PhoneNumberFormField.tsx

```diff
@@ -273,3 +260,4 @@ const CountryCodesList = ({
     </div>
   );
 };
+export default PhoneNumberFormField;
```

Comment:
Add it as part of the function itself? Like it was already written 🤔

Reply:
I am using forwordref and if I add this export default in same line it will throw an error